script.module.resolveurl/lib/resolveurl/plugins/vidspeed.py

```diff
@@ -22,8 +22,8 @@
 
 class VidSpeedResolver(ResolveGeneric):
     name = 'VidSpeed'
-    domains = ['vidspeed.cc']
-    pattern = r'(?://|\.)(vidspeed\.cc)/(?:embed-)?([0-9a-zA-Z]+)'
+    domains = ['vidspeed.cc', 'vidspeeds.com']
+    pattern = r'(?://|\.)(vidspeed(?:s)\.(?:cc|com))/(?:embed-)?([0-9a-zA-Z]+)'
```

Comment:
That regex pattern is incorrect as it will only match `vidspeeds`
Should be
`pattern = r'(?://|\.)(vidspeeds?\.(?:cc|com))/(?:embed-)?([0-9a-zA-Z]+)'`